pkg/tagger/collectors/ecs_fargate_extract.go

```diff
@@ -28,11 +29,22 @@ func (c *ECSFargateCollector) parseMetadata(meta ecs.TaskMetadata, parseAll bool
 	if meta.KnownStatus != "RUNNING" {
 		return output, fmt.Errorf("Task %s is in %s status, skipping", meta.Family, meta.KnownStatus)
 	}
+	globalTags := config.Datadog.GetStringSlice("tags")
 
 	for _, ctr := range meta.Containers {
 		if c.expire.Update(ctr.DockerID, now) || parseAll {
 			tags := utils.NewTagList()
 
+			// global tags
+			if len(globalTags) > 0 {
+				for _, value := range globalTags {
+					if strings.Contains(value, ":") {
+						tag := strings.SplitN(value, ":", 2)
+						tags.AddLow(tag[0], tag[1])
+					}
```

Comment:
we can have tags that don't contain `:` right?

```suggestion
					} else {
					    tags.AddLow(tag)
					}
```